lib/service/Dialog.dart

```diff
@@ -168,6 +168,8 @@ Dialog wrongNetworkDialog(BuildContext context) {
 
 //dynamic
 Dialog walletDialog(BuildContext context) {
+  bool isWeb = true;
+  isWeb = kIsWeb && (MediaQuery.of(context).orientation == Orientation.landscape);
```

Comment:
Is there a particular reason why you need to also check for the orientation here?

It is usually easier to build widgets for different screen sizes with a specialized widget like a responsive layout builder.
We should decide upon a good implementation for it or use one of the existing solutions out there.

You shouldn't care here whether you're on web or mobile, but rather how big/small the screen you're rendering widgets on is.

For time being this will do it but this kind of approach should be avoided in the future.

Reply:
Yeah I used this because this was used in several places of the app so I continued to implement it like that. But yeah, I would love to learn of the better way of handling responsiveness when developing the application for a variety of screen sizes.